File: practice/java/ReverseNumber.java
import java.util.Scanner;

class ReverseNumber {
    /***
     * @brief this function reverse a number
     * @param number kis number ko reverse karana hai 
     * @return int : mujhe return karana hai reverse number
     */
  static int   number(int number){
    int digit;
    int reverse = 0;
    while (number > 0) {
        
        digit = number % 10;
        reverse = (reverse *10) + digit;
        number = number / 10;
    }
    return reverse;
    }
    
public static void  main(String args[]){
    Scanner scanner = new Scanner(System.in);
    System.out.println("enter a number");
    int number = scanner.nextInt();
    int result = number(number);
    System.out.println("reverse number is:"+result);
}
}

File: practice/python/reverse_number.py
# @brief this function reverse a 
#@param number: kis number ko reverse karana hai 

def reverse_number( number):
    reverse = 0
    while(number > 0 ):
        digit = number % 10
        reverse = (reverse * 10) + digit
        number = number // 10
        result = int (reverse) 
    print(result)

print("enter a number to reverse:")
output = input()
number = int(output)
print("reverse number is:")
reverse_number(number)   
